### PR TITLE
Notable: updated links, summaries and descriptions

### PR DESCRIPTION
**Pull request by fabiospampinato**
Here's the PR I mentioned, hopefully I haven't broken anything.

I wasn't sure about what kind of formatting is enabled for the package description, so I used just some plain Markdown headers, which even if not rendered as Markdown should look ok.

### CODE REVIEW

**Comment by jerome-benoit** [comment]
Looks great. I let the package maintainer do the merge after review :)

File: notable/notable.nuspec
Comment (bspeice) [inline notable/notable.nuspec]: The `<description>` section gets re-written with whatever is in the `Readme.md` when updating; please make sure they're in sync.

**Review by bspeice — CHANGES_REQUESTED** [review]

**Comment by bspeice** [comment]
One minor change needed, but LGTM. These will all take effect on the next version of Notable that's released.

**Pull request merged**
